src/app/work-item/work-item.service.ts

```diff
@@ -1,4 +1,5 @@
-import { Injectable } from '@angular/core';
+import { Comments, Comment, CommentPost } from './../models/comment';
```

Comment:
Ha ha, about this, do you know VS code does that automatically when I try to import a package. I didn't even write this line, it's added automatically by VS Code and they follow this path style.